### PR TITLE
docs: rename category filenames from Kubernetes to Cloud Native

### LINKED ISSUE #2658: docs: "Kubernetes" docs names should be generalized to "Cloud Native"
Currently, our documentation is divided along Cloud Native and Virtual Machine administration, since these are the two broad categories. So far, we've been using "Kubernetes" instead of "Cloud Native" for those docs names, but Kubernetes is just one deployment method in a broader category of administration.

### PR DESCRIPTION
**Pull request by shmorri**
### Prepare

- [x] Read [PR guidelines](https://github.com/JanssenProject/jans/blob/main/docs/CONTRIBUTING.md#prs)
- [x] Read [license information](https://github.com/JanssenProject/jans/blob/main/LICENSE)

-------------------

### Description

Renamed most filenames from Kubernetes to Cloud Native to be more inclusive of different deployment methods.

#### Target issue
   #2658
  
  
  
closes #2658

#### Implementation Details
  

-------------------
### Test and Document the changes
- [x] Static code analysis has been run locally and issues have been fixed
- [x] Relevant unit and integration tests have been added/updated
- [x] Relevant documentation has been updated if any (i.e. user guides, installation and configuration guides, technical design docs etc)

### CODE REVIEW

**Comment by ossdhaval** [comment]
@shmorri 
- I have added a commit that adds `cloud-native` tag to the documents
- Should `admin/reference/kubernetes` change to `admin/reference/cloud-native` as well?

File: docs/admin/README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ each specific area.
 
 * **Installation** Howto's for VM and cloud native deployment
 
-* **Kubernetes Operation Guide** Day-to-day administration for cloud native
+* **Cloud Native Operation Guide** Day-to-day administration for cloud native
```
Comment (moabu) [inline docs/admin/README.md:18]: I think this is something we didn't want to move from Gluu. This created a bit of a confusion to what we actually meant by Cloud Native. As far as our docs go this is purely Kubernetes and I think the naming **should not** change.

File: docs/admin/cn-ops/README.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 tags:
   - administration
+  - cloud-native
```
Comment (moabu) [inline docs/admin/cn-ops/README.md:4]: I think adding the tag is right though, because Kubernetes is cloud native.

**Review by moabu — CHANGES_REQUESTED** [review]
I think this should not be merged. The only thing that I think should remain are the `cloud-native` tags.

**Comment by shmorri** [comment]
@ossdhaval Those reference files are Kubernetes-specific, so they definitely shouldn't change. I'm discussing the rest of it with Mohammad now.

**Pull request closed without merging**